cmake fails when gfortran is missing

Trying to build in Ubuntu 16.10 I'm getting:

-- The Fortran compiler identification is unknown
CMake Error at CMakeLists.txt:316 (enable_language):
  No CMAKE_Fortran_COMPILER could be found.

Steps to reproduce:

⟫ sudo apt-get install python3 python3-dev python3-pyqt5 python3-pyqt5.qtopengl python3-pyqt5.qtquick python3-pyqt5.qtsvg python3-numpy python3-serial python3-opengl qml-module-qtquick2 qml-module-qtquick-window2 qml-module-qtquick-layouts qml-module-qtquick-dialogs qml-module-qtquick-controls
⟫ git clone git@github.com:Ultimaker/cura-build.git
⟫ cd cura-build/
⟫ mkdir build
⟫ cd build/
⟫ cmake ..

Full cmake output: https://paste.ubuntu.com/16210321/

--- a/README.md
+++ b/README.md
@@ -106,7 +106,7 @@ Dependencies:
 To build, make sure these dependencies are installed, then clone this repository and run the following commands from your clone:
 
 ```shell
-sudo apt-get install python3 python3-dev python3-pyqt5 python3-pyqt5.qtopengl python3-pyqt5.qtquick python3-pyqt5.qtsvg python3-numpy python3-serial python3-opengl qml-module-qtquick2 qml-module-qtquick-window2 qml-module-qtquick-layouts qml-module-qtquick-dialogs qml-module-qtquick-controls
+sudo apt-get install gfortran python3 python3-dev python3-pyqt5 python3-pyqt5.qtopengl python3-pyqt5.qtquick python3-pyqt5.qtsvg python3-numpy python3-serial python3-opengl qml-module-qtquick2 qml-module-qtquick-window2 qml-module-qtquick-layouts qml-module-qtquick-dialogs qml-module-qtquick-controls
 git clone http://github.com/Ultimaker/cura-build.git
 cd cura-build
 ```

